pkg/collector/data/examples_generator.go

```diff
@@ -50,8 +50,14 @@ func ExamplesGeneration(e *Example) *collector.ResourcesContainerModel {
 	res.DomainList[0].Resources.GroupList = groupList
 
 	// add dfw
+	res.DomainList[0].Resources.SecurityPolicyList = ToPoliciesList(e.Policies)
+	res.ServiceList = getServices()
+	return res
+}
+
+func ToPoliciesList(policies []Category) []collector.SecurityPolicy {
```

Comment:
no need to export this function?

Reply:
it exported for synthesis - it is used to create the policies.
(only policies and groups are created, and added to the existing model)